### ISSUE TITLE
Error: Call to undefined function putyourlightson\blitz\helpers\array_key_first()

### ISSUE THREAD

**Issue by ademers** [putyourlightson/craft-blitz#168]
### Describe the bug
The code `{% set heroImage = entry.backgroundImage.one() %}` in a Twig template I'm attempting to cache appears to generate the following error:

```
Error
Call to undefined function putyourlightson\blitz\helpers\array_key_first()

    1. in /home/vagrant/sites/example.test/vendor/putyourlightson/craft-blitz/src/helpers/ElementQueryHelper.php at line 203
    194195196197198199200201202203204205206207208209210211212

         *
         * @return bool
         */
        public static function isOrderByRandom(ElementQuery $elementQuery): bool
        {
            if (empty($elementQuery->orderBy) || !is_array($elementQuery->orderBy)) {
                return false;
            }
     
            $key = array_key_first($elementQuery->orderBy);
     
            if (!is_string($key)) {
                return false;
            }
     
            $hasMatch = preg_match('/RAND\(.*?\)/i', $key);
     
            return (bool)$hasMatch;
        }
```

### To reproduce
Steps to reproduce the behaviour:
1. Go to page configured to be cached that contains `{% set heroImage = entry.backgroundImage.one() %}`.
2. See error message described above.
3. Comment out `{% set heroImage = entry.backgroundImage.one() %}` in template.
4. Visit page and it displays without error and is successfully cached.

### Expected behaviour
I expect to be able to use `{% set heroImage = entry.backgroundImage.one() %}` Twig code in template without error.

### Screenshots
Error: https://share.getcloudapp.com/2Nur4wJD

### Versions
- Plugin version: 3.1.1.
- Craft version: Craft Pro 3.2.10
- PHP 7.0.33

**Comment by bencroker** [comment]
Apparently I missed the fact that `array_key_first` was not yet added in PHP 7.0, doh! Adding a fix for it right now.

**Comment by ademers** [comment]
Thanks, @bencroker !

**Comment by bencroker** [comment]
Fixed in 3.2.0.
